CI/integration_tests/calculators/test_green_kubo_self_diffusion_coefficients.py

```diff
@@ -71,9 +72,9 @@ def test_calculator(tmp_path):
     os.chdir(tmp_path)
     project = mds.Project()
     # introduce nontrivial units to make sure all conversions are correct
-    units = mdsuite.utils.units.si
-    units.length = 0.000007654
```

Comment:
A general comment here, should this just be a custom unit declaration rather than changing SI units?

Reply:
I just modified the original test. I guess it could also be a new unit but using this method we can show how to "inherit" from an existing one if this might ever be necessary.

Reply:
Yeah I saw it is from the old test, sure we can leave it